src/runtime/virtcontainers/qemu_amd64.go

```diff
@@ -227,7 +227,7 @@ func (q *qemuAmd64) enableProtection() error {
 		virtLog.WithFields(logrus.Fields{
 			"subsystem":     "qemuAmd64",
 			"machine":       q.qemuMachine,
-			"kernel-params": q.kernelParameters}).
+			"kernel-params": q.kernelParameters()}).
```

Comment:
A few lines below we do the same for the sevProtection.
I'm also afraid we me do the same for other arches, and it's worth checking and fixing all the cases at once.

Reply:
Okay, other arches are fine, but we should also fix the case below for the sevProtection.

Reply:
Sorry I missed that line.

Reply:
@fidencio It's hard to call `archBase.kernelParameters()`, because it needs a debug flag parameter, but qemu_amd64 doesn't hold one, so I changed it to print all the parameters fields.